packages/@office-iss/react-native-win32/metro.config.js

```diff
@@ -12,6 +12,12 @@ const rnwTesterPath = fs.realpathSync(
     require.resolve('@office-iss/react-native-win32-tester/package.json'),
   ),
 );
+const virtualizedListPath = fs.realpathSync(
```

Comment:
RNW consumers should not need to add this. It is just because Metro does not like workspace packages in our dev environment.

Reply:
Maybe add this as a comment.

Reply:
Added in each of the config files